src/nestjs-back/src/channels/channels.controller.ts

```diff
@@ -0,0 +1,85 @@
+import {
+    Body,
+    Controller,
+    Delete,
+    Get,
+    Param,
+    Patch,
+    Post
+} from '@nestjs/common';
+import { ChannelsService } from './channels.service';
+import { CreateChannelDto } from './dto/create-channel.dto';
+import { UpdateChannelDto } from './dto/update-channel.dto';
+
+@Controller('channels')
+export class ChannelsController {
+    constructor(
+        private readonly channelsService: ChannelsService
+    ) {}
+
+    @Get()
+    findAll() {
+        return this.channelsService.findAll();
+    }
+
+    @Get('/owner/:ownerId')
+    findAllByOwner(@Param('ownerId') ownerId: string) {
```

Comment:
Not very RESTful, filtering is usually done through the use of query params
For example:
`/channels?owner_id=345`
`/channels?visibility=public&owner_id=345`

Creating controller routes for that purpose is rarely seen and, from what I know, isn't really a RESTful practice.